test/integration/patients/worklist/worklist.js

```diff
@@ -2207,13 +2193,19 @@ context('worklist page', function() {
       .type(formatDate(testTsSubtract(1, 'month'), 'MMM D'));
 
     cy
-      .get('[data-select-all-region]')
-      .find('.fa-check-square');
+      .get('@flowList')
+      .find('.work-list__item')
+      .contains('Test Flow');
 
     cy
-      .get('[data-filters-region]')
-      .find('.js-bulk-edit')
-      .should('contain', 'Edit 3 Flows');
```

Comment:
I do think we _could_ test that this has the correct number by not introducing fixture data that could randomly break particular tests.

Reply:
Yeah, there was no need to dynamic date in this test. Made them static.